Source/WebKit/Shared/API/glib/WebKitContextMenuItem.cpp

```diff
@@ -333,6 +333,28 @@ GAction* webkit_context_menu_item_get_gaction(WebKitContextMenuItem* item)
 #endif // ENABLE(CONTEXT_MENUS)
 }
 
+/**
+ * webkit_context_menu_item_get_title:
+ * @item: a #WebKitContextMenuItem
+ *
+ * Gets the title associated to @item.
+ *
+ * Returns: (transfer none): the title associated to the #WebKitContextMenuItem,
+ *    or %NULL if @item is a separator.
+ *
+ * Since: 2.18
+ */
+const char* webkit_context_menu_item_get_title(WebKitContextMenuItem* item)
+{
+    g_return_val_if_fail(WEBKIT_IS_CONTEXT_MENU_ITEM(item), nullptr);
+
+#if ENABLE(CONTEXT_MENUS)
+    return item->priv->menuItem->title().utf8().data();
```

Comment:
This is returning a value that is released right after the function finishes, because String::utf8() returns a temporary. We need to cache the CString and returns its data to keep this returning a const char

Reply:
I've decided to just go with reassigning it every call. If a more complicated solution is desired that checks if the CString is still accurate to avoid re-conversion, please let me know.